tests/data/composition.py

```diff
@@ -31,13 +31,15 @@ def test(self) -> None:
                         # Another
                     ):
                         print(i)
-        xxxxxxxxxxxxxxxx = Yyyy2YyyyyYyyyyy(
-            push_manager=context.request.resource_manager,
-            max_items_to_push=num_items,
-            batch_size=Yyyy2YyyyYyyyyYyyy.FULL_SIZE,
-        ).push(
-            # Only send the first n items.
-            items=items[:num_items]
+        xxxxxxxxxxxxxxxx = (
```

Comment:
I'm not sure about this one. I slightly prefer the old one

Reply:
Yes, the extra set of parentheses doesn't really help anything here.